### DISCUSSION TITLE
Felicidad menor igual que 0

### DISCUSSION THREAD

**Discussion by sbarriag** [IIC2133-PUC/2024-2#111] [Tarea 1] [answered]
Hola, podemos asumir que cuando haya un `Happiness Buff` nunca se va a sumar un número negativo?? es decir **increase** siempre será mayor o igual que 0? o hay que hacer una verificación manual por nuestra cuenta?
Muchas gracias

**Comment by sebacarrascop** [comment, marked as answer]
hola, siempre se sumará un número positivo.
